srcpkgs/mozo/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'mozo'
 pkgname=mozo
-version=1.26.1
-revision=2
+version=1.26.2
```

Comment:
1.27.0 is available

Reply:
odd numbered versions are considered as development versions for mate and not official releases